_events/sr2025/kickstart.md

```diff
@@ -32,20 +32,21 @@ attendance at Kickstart.
 | Time  | Info |
 |-------|------|
 | 11:00 | Introductory presentation, where the game of SR2025 will be announced. |
-| 12:30 | Lunch. |
+| 12:30 | There will be a lunch break. Teams should bring their own lunches or get them from food vendors near the venue. |
 | 13:30 | Microgames, a set of tasks to familiarise you with the kit. |
 | 17:00 | Finish. |
 
 ## What to bring
 
 * Your robot kit (we intend to ship kits to teams ahead of Kickstart)
 * Any tools which might be useful to work with your kit
+* A laptop to program your robots with
```

Comment:
```suggestion
* A laptop to program your robots with. Ideally with some way of plugging in a [USB Type-A plug](https://www.viewsonic.com/library/tech/usb-c-usb-b-and-usb-a-whats-the-difference/#USB_The_Basics)
```

Reply:
Thanks for raising, 68b9ab6957a6b756790fc0cd273be34dd4e9ff7a.